x-pack/plugins/security_solution/cypress/tasks/login.ts

```diff
@@ -78,8 +78,13 @@ const loginViaEnvironmentCredentials = () => {
   // programmatically authenticate without interacting with the Kibana login page
   cy.request({
```

Comment:
Hey @elastic/siem, can anyone please let me know what this code does? Is it only for tests or you use this for anything else? We're changing our internal login endpoint and I want to make sure I properly adjusted your code that depends on it.

Reply:
Correct @azasypkin, this is just for our e2e cypress tests. These tests run as part of CI, so we should see a failure then if there are any issues. Thanks for the ping! 🙂 

cc @MadameSheema

Reply:
> Correct @azasypkin, this is just for our e2e cypress tests. These tests run as part of CI, so we should see a failure then if there are any issues. Thanks for the ping! slightly_smiling_face

Thanks for confirming! `Security solution Cypress Tests` suite is green, so we should be good.